docs/source/iati.rst

```diff
@@ -6,12 +6,83 @@ Subpackages
 
 .. toctree::
 
-    iati.core
     iati.tests
 
 Submodules
 ----------
 
+iati\.codelists module
```

Comment:
In `docs/source/iati.core.tests.rst`, the `\` characters have been removed, so not sure if the approach here is consistent...?

Reply:
Also applies to other instances of submodule headings and instances in `docs/source/iati.tests.rst`.

Reply:
Running `make docs` should fix this. If not, delete `docs/build/`, then re-run this.

Reply:
...actually this is in `docs/source/` - just run `make docs` and make sure any `iati.core` references have said turrah.

Reply:
This was resolved in ce3b5e1.